libraries/LCD_SharpBoosterPack_SPI/LCD_SharpBoosterPack_SPI.cpp

```diff
@@ -576,29 +576,34 @@ static void SendToggleVCOMCommand(UArg arg)
         unsigned char command = SHARP_LCD_CMD_CHANGE_VCOM;
         command |= VCOMbit;                    //COM inversion bit
 
-        if (LCDPowerMode == LCDPowerSaveOn)
```

Comment:
Why does this not work for the CC13x0? I rather fix the underlying code than put in a one architecture only patch.